import-automation/progress-dashboard-rest/app.yaml

```diff
@@ -0,0 +1,4 @@
+runtime: python37
+entrypoint: gunicorn -b :$PORT app.main:FLASK_APP
```

Comment:
How did we choose gunicorn?  What services does it provide for us?

Reply:
gunicorn is an application server that serves the flask app. This https://serverfault.com/questions/331256/why-do-i-need-nginx-and-something-like-gunicorn may be helpful.